Majasdarbs/05-majasdarbs/05-majasdarbs/switch.js

```diff
@@ -0,0 +1,24 @@
+const mdarbaDiena = "Pirmdiena";
+
+switch (darbaDiena) {
+    case "Pirmdiena":
+    console.log("1");
+    break;
+    case "Otrdiena":
+    console.log("2");
+    break;
+    case "Trešdiena":
+    console.log("3");
+    break;
+    case "Ceturtdiena":
+    console.log("4");
+    break;
+    case "Piektdiena":
+    console.log("5");
+    break;
+    case "Sestdiena":
+    console.log("6");
+    break;
+    case "Svētdiena":
+    console.log("7");
+    break; }
```

Comment:
you need to use default case in switches https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch